backend/api/src/Routes/routes.go

```diff
@@ -37,14 +37,19 @@ func SetupRoutes(gormObj *gorm.DB) {
 	r.GET("/api/list-due-exams", listDueExams)
 	// auth protected routes
 	auth := r.Group("/", authMiddleware)
-	auth.GET("/api/auth-hello", hello)
-	auth.GET("/api/list-exam-users", listExamUsers) // TODO: should be adminMiddleware protected
-	auth.POST("/api/create-user", createUser)       // TODO: should be adminMiddleware protected
-	auth.POST("/api/create-exam", createExam)       // TODO: should be adminMiddleware protected
-	auth.POST("/api/delete-exam", deleteExam)       // TODO: should be adminMiddleware protected
-	auth.POST("/api/register-exam", registerToExam)
-	auth.POST("/api/unregister-exam", unregisterFromExam)
-	auth.POST("/api/add-user-expo-push-token", addUserExpoPushToken)
+	{
+		auth.GET("/api/auth-hello", hello)
+		auth.POST("/api/register-exam", registerToExam)
+		auth.POST("/api/unregister-exam", unregisterFromExam)
+		auth.POST("/api/add-user-expo-push-token", addUserExpoPushToken)
+		adminAuth := r.Group("/admin", adminMiddleware)
```

Comment:
```suggestion
		adminAuth := auth.Group("/admin", adminMiddleware)
```

should be nested under auth.

Reply:
big boi fixed it.